docs/sources/mimir/configure/configure-otel-collector.md

```diff
@@ -10,14 +10,17 @@ weight: 150
 # Configure the OpenTelemetry Collector to write metrics into Mimir
 
 {{% admonition type="note" %}}
-To send OTLP data to Grafana Cloud, refer to [Send data using OpenTelemetry Protocol (OTLP)](https://grafana.com/docs/grafana-cloud/send-data/otlp/send-data-otlp/).
+To send OTel data to Grafana Cloud, refer to [Send data using OpenTelemetry Protocol (OTLP)](https://grafana.com/docs/grafana-cloud/send-data/otlp/send-data-otlp/).
 {{% /admonition %}}
 
 When using the [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/), you can write metrics into Mimir via two options: `prometheusremotewrite` and `otlphttp`.
 
-We recommend using the `prometheusremotewrite` exporter when possible because the remote write ingest path is tested and proven at scale.
+We recommend using each protocol's respective exporter and native Mimir endpoint:
```

Comment:
```suggestion
As a best practice, use each protocol's respective exporter, as well as the following Mimir endpoints:
```

Reply:
I tried to clear this up a bit. LMK is the meaning is preserved, or if we should try something else.

Reply:
I would leave this as is. "We recommend" is the language we use elsewhere in the docs, and I think "Best Practice" implies that a lot more research and consensus than we want to imply here.

Reply:
also the endpoints aren't explicitly listed here, those names are for the exporters.
The endpoints are expanded on in each corresponding section below.
We could drop the bit about the endpoints completely, but saying "the following Mimir endpoints" would not make sense.

Reply:
Thanks for clarifying. Let's leave as it is 😄